### DISCUSSION TITLE
LB SERVER

### DISCUSSION THREAD

**Discussion by JhonyBravoX** [masoudgb/Xtream-Ui#3] [Q&A]
I want to install LB on the same server. What server ID should I provide? How to do it? Also where to get ort URL to stream in IPTV app.

**Comment by masoudgb** [comment]
> I want to install LB on the same server. What server ID should I provide? How to do it? Also where to get ort URL to stream in IPTV app.

good time I did not understand your question The lb server is installed as a balancer on the second server and not on the main server. To install on the lb server, run the script and enter the information
